backend/ErrorHandler.js

```diff
@@ -0,0 +1,8 @@
+const ERRORS = {
```

Comment:
So i would say a better name for the file is Errors under the enums folder. so 
enums/Errors.js

Reply:
reason is Error handler means you have a handler inside which we do not have.